app/controllers/teacher_interface/further_information_requests_controller.rb

```diff
@@ -8,6 +8,17 @@ def show
     def edit
     end
 
+    def update
+      if @view_object.can_submit?
```

Comment:
I was wondering about this in the submit application flow. If we're going to check here then we should do something different if it returns false. Given we're only showing the button if this is true then this is probably redundant. If we wanted an extra layer of protection maybe we should raise in the service if this is false.

Reply:
That makes sense, happy to make the service raise an exception.

Reply:
I've amended the commit to implement it with an exception.